### ISSUE TITLE
Destroying a session does not remove it from cache

### ISSUE THREAD

**Issue by aaronurkin** [battlejj/express-mongoose-store#10]
Destroying a session 'cache.del' method must be invoked to remove session data from cache.

![something like this](https://user-images.githubusercontent.com/36369007/36074063-9ce8f8ba-0f4b-11e8-8a04-f978904d311a.png)

**Comment by alanosman** [comment]
Yup - I had a problem where we could not destroy the session and the user would not log out.  Went hunting and found this from @aaronurkin (thanks), and it fixed it.
